libs/langchain/langchain/document_loaders/recursive_url_loader.py

```diff
@@ -69,19 +73,30 @@ def get_child_links_recursive(
         response = requests.get(url)
         soup = BeautifulSoup(response.text, "html.parser")
         all_links = [link.get("href") for link in soup.find_all("a")]
-
-        # Extract only the links that are children of the current URL
-        child_links = list(
-            {
-                link
-                for link in all_links
-                if link and link.startswith(current_path) and link != current_path
-            }
-        )
-
+        child_links = set()
```

Comment:
Still returns 0 links for this url `https://js.langchain.com/docs/modules/memory/integrations/` w/ `crawl_siblings =True`. odd. did not dig in further, but can confirm the updated loader is working at least for that particular link:

https://github.com/langchain-ai/langchain/pull/8502